kolibri/plugins/learn/assets/src/views/points-slidein/index.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+
+  <div class="points-slidein">
+    <div class="points">
+      <points-icon class="points-icon" :active="true"/>
+      <span class="points-amount">{{ $tr('plusPoints', { maxPoints }) }}</span>
+    </div>
+
+    <ui-icon-button
+      type="secondary"
+      color="default"
+      size="small"
+      icon="close"
+      :ariaLabel="$tr('dismiss')"
+      @click="$emit('close')"
+    />
+    </div>
+
+</template>
+
+
+<script>
+
+  import { contentPoints } from 'kolibri.coreVue.vuex.getters';
+  import { MaxPointsPerContent, ContentNodeKinds } from 'kolibri.coreVue.vuex.constants';
+  import pointsIcon from 'kolibri.coreVue.components.pointsIcon';
+  import uiIconButton from 'keen-ui/src/UiIconButton';
+
+  export default {
+    $trNameSpace: 'pointsSlidein',
+    $trs: {
+      plusPoints: '+ { maxPoints, number } Points',
```

Comment:
As this string is identical to the one in the points popup, this should be fine as is.